### PR TITLE
fix xvfb passing multiple parameters

### PR DESCRIPTION
**Pull request by SanFenZuiCom**
fix flagXVFB does not add to xvfb-run command, and add multiple parameters.

### CODE REVIEW

**Review by ysmood — CHANGES_REQUESTED** [review]
Please fix the CI error.

Please update this method:

https://github.com/go-rod/rod/blob/47eeb7a4e5573942abcf38a5e8a3ce7445bdd936/lib/launcher/launcher.go#L215-L218/

Make it something like:

```go
func (l *Launcher) XVFB(args string...) *Launcher {
```

**Comment by BenjaminB64** [comment]
I submit a PR on your master @SanFenZuiCom https://github.com/SanFenZuiCom/rod/pull/1

**Comment by SanFenZuiCom** [comment]
> I submit a PR on your master @SanFenZuiCom [SanFenZuiCom#1](https://github.com/SanFenZuiCom/rod/pull/1)

Thanks a lot.There seems to be something wrong with my git merge.

File: lib/examples/remote-launch/main.go [hunk 1]
```diff
@@ -2,7 +2,6 @@ package main
 
 import (
 	"fmt"
-
```
Comment (ysmood) [inline lib/examples/remote-launch/main.go:5]: Don't remove this line

File: lib/examples/remote-launch/main.go [hunk 2]
```diff
@@ -18,13 +17,12 @@ func main() {
 	//
 	// For more information, check the doc of launcher.RemoteLauncher
 	l := launcher.MustNewRemote("")
-
```
Comment (ysmood) [inline lib/examples/remote-launch/main.go:21]: Don't remove this line

File: lib/examples/remote-launch/main.go [hunk 3]
```diff
@@ -18,13 +17,12 @@ func main() {
...
+	l.NoSandbox(true)
+	l.XVFB("--server-num=5","--server-args=-screen 0 1600x900x16")
 	// Manipulate flags like the example in examples_test.go
 	l.Set("any-flag").Delete("any-flag")
-
 	// Launch with headful mode
-	l.Headless(false).XVFB()
-
```
Comment (ysmood) [inline lib/examples/remote-launch/main.go:27]: Don't remove this line

File: lib/examples/remote-launch/main.go [hunk 4]
```diff
@@ -18,13 +17,12 @@ func main() {
 	//
 	// For more information, check the doc of launcher.RemoteLauncher
 	l := launcher.MustNewRemote("")
-
+	l.NoSandbox(true)
```
Comment (ysmood) [inline lib/examples/remote-launch/main.go:20]:
```suggestion
	l.NoSandbox(true)
```

File: lib/examples/remote-launch/main.go [hunk 5]
```diff
@@ -18,13 +17,12 @@ func main() {
...
+	l.XVFB("--server-num=5","--server-args=-screen 0 1600x900x16")
 	// Manipulate flags like the example in examples_test.go
 	l.Set("any-flag").Delete("any-flag")
-
 	// Launch with headful mode
-	l.Headless(false).XVFB()
-
+	l.Headless(false)
```
Comment (ysmood) [inline lib/examples/remote-launch/main.go:25]:
```suggestion
	// Manipulate flags like the example in examples_test.go
	l.Set("any-flag").Delete("any-flag")

	// Launch with headful mode
	l.Headless(false).XVFB("--server-num=5", "--server-args=-screen 0 1600x900x16")
```

**Review by ysmood — CHANGES_REQUESTED** [review]
There should be no changes for files like "go.mod".

Google "rebase upstream master"

File: lib/examples/remote-launch/main.go [hunk 6]
```diff
@@ -21,10 +21,8 @@ func main() {
 
 	// Manipulate flags like the example in examples_test.go
 	l.Set("any-flag").Delete("any-flag")
-
 	// Launch with headful mode
-	l.Headless(false).XVFB()
-
```
Comment (ysmood) [inline lib/examples/remote-launch/main.go:27]: Don't remove this space line

File: lib/examples/remote-launch/main.go [hunk 7]
```diff
@@ -21,10 +21,8 @@ func main() {
 
 	// Manipulate flags like the example in examples_test.go
 	l.Set("any-flag").Delete("any-flag")
-
```
Comment (ysmood) [inline lib/examples/remote-launch/main.go:24]: Don't remove this space line

File: go.sum [hunk 8]
```diff
@@ -4,7 +4,7 @@ github.com/ysmood/got v0.12.0 h1:Ol4cpy6Xdq1KCjPlWSA+tvekrnt9cV6LIw+Jvx0dj4M=
...
 github.com/ysmood/gotrace v0.2.2 h1:006KHGRThSRf8lwh4EyhNmuuq/l+Ygs+JqojkhEG1/E=
 github.com/ysmood/gotrace v0.2.2/go.mod h1:TzhIG7nHDry5//eYZDYcTzuJLYQIkykJzCRIo4/dzQM=
-github.com/ysmood/gson v0.6.3 h1:4cU+5oOdsyundXHy00t99H0rLXLthuseD3x6W+xmCiU=
-github.com/ysmood/gson v0.6.3/go.mod h1:3Kzs5zDl21g5F/BlLTNcuAGAYLKt2lV5G8D1zF3RNmg=
-github.com/ysmood/leakless v0.6.16 h1:PVS0EY7eFLzkUTqVJbST7cLz9V+58L+4TIlgxmCLi3k=
-github.com/ysmood/leakless v0.6.16/go.mod h1:R8iAXPRaG97QJwqxs74RdwzcRHT1SWCGTNqY8q0JvMQ=
+github.com/ysmood/gson v0.6.4 h1:Yb6tosv6bk59HqjZu2/7o4BFherpYEMkDkXmlhgryZ4=
+github.com/ysmood/gson v0.6.4/go.mod h1:3Kzs5zDl21g5F/BlLTNcuAGAYLKt2lV5G8D1zF3RNmg=
+github.com/ysmood/leakless v0.7.0 h1:XCGdaPExyoreoQd+H5qgxM3ReNbSPFsEXpSKwbXbwQw=
+github.com/ysmood/leakless v0.7.0/go.mod h1:R8iAXPRaG97QJwqxs74RdwzcRHT1SWCGTNqY8q0JvMQ=
```
Comment (ysmood) [inline go.sum:10]:
```suggestion
```

File: go.mod [hunk 9]
```diff
@@ -6,6 +6,6 @@ require (
 	github.com/ysmood/goob v0.3.0
 	github.com/ysmood/got v0.12.0
 	github.com/ysmood/gotrace v0.2.2
-	github.com/ysmood/gson v0.6.3
-	github.com/ysmood/leakless v0.6.16
+	github.com/ysmood/gson v0.6.4
+	github.com/ysmood/leakless v0.7.0
```
Comment (ysmood) [inline go.mod:10]:
```suggestion
```

File: lib/launcher/launcher.go [hunk 10]
```diff
@@ -42,7 +42,7 @@ type Launcher struct {
 // New returns the default arguments to start browser.
 // Headless will be enabled by default.
 // Leakless will be enabled by default.
-// UserDataDir will use OS tmp dir by default.
+// UserDataDir will use OS tmp dir by default, this folder will usually be cleaned up by the OS after reboot.
```
Comment (ysmood) [inline lib/launcher/launcher.go:45]:
```suggestion
```

**Review by ysmood — CHANGES_REQUESTED** [review]

**Comment by ysmood** [comment]
Too many problems, I have to merge it manually.

Merged on https://github.com/go-rod/rod/commit/b72ea332d7b54820e442bc2244f91f65b7beae31

**Pull request closed without merging**